janus/vim/vimrc

```diff
@@ -27,6 +27,8 @@ call janus#add_group("colors")
 
 if filereadable(expand("~/.vimrc.before"))
   source ~/.vimrc.before
+else
+  let g:syntastic_mode_map = { 'mode': 'passive' }
```

Comment:
@kalbasit thanks for the insight on preferred location. I'll move it there and add a note to the README.md.